From 6fdf85cb284df41384ab3732ae88b6ea3b04e17c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Arthur=20Exalta=C3=A7=C3=A3o?=
 <arthur.exaltacao@chainguard.dev>
Date: Mon, 18 Aug 2025 15:53:32 -0300
Subject: [PATCH] freeipa: ignore last element of setuptools version
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This is a workaround since our setuptools package seems to be adding
`post(date)` to the version on python, which is unexpected by this
script

Signed-off-by: Arthur Exaltação <arthur.exaltacao@chainguard.dev>
---
 ipasetup.py.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ipasetup.py.in b/ipasetup.py.in
index 56a1f3b06..b1951f0f9 100644
--- a/ipasetup.py.in
+++ b/ipasetup.py.in
@@ -61,7 +61,7 @@ import setuptools
 
 VERSION = '@VERSION@'
 
-SETUPTOOLS_VERSION = tuple(int(v) for v in setuptools.__version__.split("."))
+SETUPTOOLS_VERSION = tuple(int(v) for v in setuptools.__version__.split(".")[:-1])
 
 PACKAGE_VERSION = {
     'cryptography': 'cryptography >= 1.6',
-- 
2.50.1


